demo-extended/values.schema.json

```diff
@@ -74,6 +79,106 @@
         "resources"
       ]
     },
+    "probes": {
+      "type": "object",
+      "title": "Probes",
+      "description": "Configuration for container probes",
+      "properties": {
+        "enabled": {
+          "title": "Enabled",
+          "type": "boolean",
+          "default": true,
+          "description": "Toggle to enable or disable all probes"
+        },
+        "startup": {
+          "title": "Startup Probe",
+          "type": "object",
+          "description": "Configuration for the startup probe",
+          "properties": {
+            "path": {
+              "title": "Path for probe",
+              "type": "string",
+              "description": "The endpoint to probe for startup readiness"
+            },
+            "initialDelaySeconds": {
+              "title": "Initial Delay Seconds",
+              "type": "integer",
+              "description": "The number of seconds to wait before performing the first probe"
+            },
+            "periodSeconds": {
+              "title": "Period Seconds",
+              "type": "integer",
+              "description": "The number of seconds between each probe"
+            },
+            "failureThreshold": {
+              "title": "Failure Threshold",
+              "type": "integer",
+              "description": "The number of consecutive failures required to consider the probe as failed"
+            }
+          }
+        },
+        "liveness": {
+          "title": "Liveness Probe",
+          "type": "object",
+          "description": "Configuration for the liveness probe",
+          "properties": {
+            "path": {
+              "title": "Path for probe",
+              "type": "string",
+              "description": "The endpoint to probe for liveness"
+            },
+            "initialDelaySeconds": {
+              "title": "Initial Delay Seconds",
+              "type": "integer",
+              "description": "The number of seconds to wait before performing the first probe"
+            },
+            "periodSeconds": {
+              "title": "Period Seconds",
+              "type": "integer",
+              "description": "The number of seconds between each probe"
+            },
+            "failureThreshold": {
+              "title": "Failure Threshold",
+              "type": "integer",
+              "description": "The number of consecutive failures required to consider the probe as failed"
+            }
+          }
+        },
+        "readiness": {
```

Comment:
Can you add `order`: ["path", "initialDelaySeconds"...]

Reply:
Hey @petar-cvit , Yup its done now. Please Review it :)